puzzle/plugins/vcf/mixins/variant_mixin.py

```diff
@@ -160,7 +160,7 @@ def variants(self, case_id, skip=0, count=1000, filters=None):
             else:
                 skip_index += 1
         
-        return Results(iter(result), len(result))
+        return Results(result, len(result))
```

Comment:
Yes it was stupid to convert the list to a iterator in the first place, we already store it in memory